covidthailand.py

```diff
@@ -2817,7 +2909,7 @@ def save_plots(df):
         y=cols,
         kind="area",
```

Comment:
@flyingvince we can convert all these to using plot_area right?

Reply:
I think so, yes. Let's agree on the `plot_area` function arguments and then I can convert to using that function for all visualisations.

Reply:
ok. but it might be the case of discovering whats needed when converting them one by one?
I know cols and legend are both missing. might be other things needed?

Reply:
Let me try to convert the process of creating the .png files gradually. Let's see what else is missing.